json-schemas/permissions/scopes.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json",
+  "type": "object",
+  "definitions": {
+    "protocols-configure-scope": {
+      "type": "object",
+      "properties": {
+        "interface": {
+          "const": "Protocols"
+        },
+        "method": {
+          "const": "Configure"
+        }
+      }
+    },
+    "protocols-query-scope": {
+      "type": "object",
+      "properties": {
+        "interface": {
+          "const": "Protocols"
+        },
+        "method": {
+          "const": "Query"
+        }
+      }
+    },
+    "records-delete-scope": {
+      "type": "object",
+      "required": [
+        "interface",
+        "method"
+      ],
+      "properties": {
+        "interface": {
+          "const": "Records"
+        },
+        "method": {
+          "const": "Delete"
+        },
+        "recordIds": {
```

Comment:
The idea of granting someone the ability to delete Record X, Y, and Z seems strange to me.

Reply:
https://github.com/TBD54566975/dwn-sdk-js/pull/448#discussion_r1282442605